client/src/components/RatingsAndReviews/AddReviewForm.jsx

```diff
@@ -0,0 +1,418 @@
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import AddCharacteristic from './AddCharacteristic';
+
+const AddReviewForm = class extends React.Component {
```

Comment:
I was thrown off for a second because I wasn't using to seeing it this way, instead of ' class AddReviewForm extends React.Component ' , but thanks JS for having multiple ways of creating variables and classes XD

Reply:
This is linter's work, read it in an ESLint helper page